src/main/resources/application.conf

```diff
@@ -0,0 +1,19 @@
+sqs2kinesis {
+  sqs-endpoint = "http://0.0.0.0:4576"
+  sqs-endpoint = ${?SQS_ENDPOINT}
+  sqs-queue    = "http://0.0.0.0:4576/queue/good-events-queue"
+  sqs-queue    = ${?SQS_QUEUE}
+  kinesis-endpoint   = "http://0.0.0.0:4568/"
+  kinesis-endpoint   = ${?KINESIS_ENDPOINT}
```

Comment:
Presumably in production we are using the default SQS and Kinesis endpoints - is it still required to supply these in that case?

Reply:
I treat it like a standard solution, to have some default option and optional overwrite by env vars. It's true that it might be overkill here, as during deployment these are not used, but the configs from TF code. On the other hand, removing it completely makes it hard to use it without our TF code. In TF code changes I do use the env vars to set up the queues and stream (https://github.com/snowplow-devops/terraform-modules/pull/991/files#diff-5b41274a289ce949c2a6dcfe89ed49a3R254), so I can remove the "local" settings and leave non optional env vars settings. 
Thanks for spotting it!